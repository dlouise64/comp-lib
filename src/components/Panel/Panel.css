.panel {
}

.panel.success header {
  background: rgb(167, 225, 180);
  padding: 7px 25px;
  font-size: 24px;
  font-weight: bold;
  color: #333;
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
}

.panel.success div {
  background: rgba(167, 225, 180, 0.15);
  padding: 15px 25px;
  border-bottom-left-radius: 4px;
  border-bottom-right-radius: 4px;
}


