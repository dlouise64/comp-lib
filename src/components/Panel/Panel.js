import React from 'react';
// import PropTypes from 'prop-types';

import './Panel.css';

class Panel extends React.Component {
  render() {
    return (
      <div className="panel">Panel</div>
    );
  }
}

Panel.propTypes = {
};

Panel.defaultProps = {
}

export default Panel;
