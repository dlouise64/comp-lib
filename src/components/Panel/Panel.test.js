import React from 'react';
import renderer from 'react-test-renderer';
import Panel from './Panel';

test('<Panel /> renders correctly', () => {
  const tree = renderer.create(<Panel
    title="Panel title"
    theme="success"
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
