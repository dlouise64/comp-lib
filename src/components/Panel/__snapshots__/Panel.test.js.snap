// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Panel /> renders correctly 1`] = `
<div
  className="panel"
>
  Panel
</div>
`;
